src/lib/samples.rs

```diff
@@ -19,6 +19,9 @@ pub struct Sample {
     pub sample_id: String,
     /// DNA barcode associated with the sample
     pub barcode: String,
+    /// DNA barcode as a byte
+    #[serde(skip_deserializing)]
+    pub barcode_bytes: Vec<u8>,
```

Comment:
Question: Why add a `barcode_bytes` field?

`String` implements `Clone` and `into_bytes` so if a user needs bytes they can still call `sample.barcode.clone().into_bytes()` if they need a `Vec<u8>`. And if they need a slice they can use `sample.barcode.as_bytes()`.

Reply:
I'm getting confused about parts of a previous implementation that has since been removed.